packages/passport/sdk/src/guardian/index.ts

```diff
@@ -228,13 +233,13 @@ export default class GuardianClient {
     }
   }
 
-  private async evaluateMessage(
-    { chainID, payload }:GuardianMessageValidationParams,
+  private async handleEIP712MessageEvaluation(
+    { chainID, payload }:GuardianEIP712MessageEvaluationParams,
   ): Promise<guardian.MessageEvaluationResponse> {
     try {
       const user = await this.authManager.getUserZkEvm();
       if (user === null) {
-        throw new PassportError('evaluateMessage requires a valid ID token or refresh token. Please log in first', PassportErrorType.NOT_LOGGED_IN_ERROR);
+        throw new PassportError('handleEIP712MessageEvaluation requires a valid ID token or refresh token. Please log in first', PassportErrorType.NOT_LOGGED_IN_ERROR);
```

Comment:
This error (along with the equivalent ERC191 error below) are likely to be surfaced to consumers of Passport. Given that consumers aren't aware of these functions (`handleEIP712MessageEvaluation` and `handleERC191MessageEvaluation`), can we update the error messages to be a bit more generic please?